build.gradle

```diff
@@ -139,32 +143,11 @@ task sourcesJar(type: Jar, dependsOn: classes) {
   from sourceSets.main.allSource
 }
 
-task testsJar(type: Jar) {
-  group = 'Publications'
-  description = 'Create jar of tests.'
-  archiveClassifier = 'tests'
-  from sourceSets.test.output
-}
-
-task reportsZip(type: Zip, dependsOn: check) {
-  group = 'Publications'
-  description = 'Create a zip of all reports.'
-  archiveClassifier = 'reports'
-  from reporting.baseDir
-}
-
-// Local published to ~/.m2 - mavenLocal() for testing
 publishing {
-  repositories {
-    mavenLocal()
-  }
```

Comment:
This is useful for local builds to test via `examples`. Let's keep that for ourselves.

Reply:
This still works and is added by default.

I tested via:

```
gradlew build -x test -s
gradlew publishToMavenLocal -x test -s
```